src/main/java/com/fasterxml/jackson/databind/introspect/POJOPropertiesCollector.java

```diff
@@ -106,6 +106,11 @@
     
     protected LinkedList<AnnotatedMember> _anySetterField;
 
+    /**
+     * Method(s) annotated with 'JsonKey' annotation
```

Comment:
Method(s) and/or Field(s)?

Reply:
I copied the comment here from `jsonValueAccessors`. I'll update them both to match the better language in `JsonValue`.